docs/guide/developer-guide/intermediate-integration/front-end-application.md

```diff
@@ -3,3 +3,451 @@
+       -  ```webpack.common.js```: Common configuration for [Webpack](https://webpack.js.org/concepts/) needed to package the application 
+       -  ```webpack.dev.js``` : Development configuration of the application for Webpack, needed to simplify developing and debugging the application
+       -  ```webpack.prod.js```:Production configuration of the application for Webpack, used to build final package deployable on production
+2.  Here is the list of files included that are the extension : 
+   -  ```src/main/webapp/META-INF/exo-conf/configuration.xml``` : extension activator configuration, more details in [Prepare extension project](/guide/developer-guide/prepare-extension-project)
+   -  ```src/main/webapp/WEB-INF/web.xml``` : Web application descriptor. it contains 
+      -  PortalContainerConfigOwner listener to register the current war as an eXo extension 
+      -  ResourceRequestFilter filter to add CSS/JS resources and make them retrieved as declared in gatein-resources.xml file
+      ```xml
+         <?xml version="1.0" encoding="ISO-8859-1" ?>
+          <web-app
+                  version="3.0"
+                  metadata-complete="true"
+                  xmlns="http://java.sun.com/xml/ns/javaee"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
+            <!-- display name should be the name of the war file. It will be used as the application name -->
+            <display-name>vue-webpack-sample</display-name>
+            <absolute-ordering/>
+
+            <!-- This web listener is used to register the current war in the list of deployed extension files -->
+            <listener>
+              <listener-class>org.exoplatform.container.web.PortalContainerConfigOwner</listener-class>
+            </listener>
+
+            <!-- This web filter will load static CSS/JS files according to the declaration in the gatein-resources.xml file -->
+            <filter>
+              <filter-name>ResourceRequestFilter</filter-name>
+              <filter-class>org.exoplatform.portal.application.ResourceRequestFilter</filter-class>
+            </filter>
+
+            <filter-mapping>
+              <filter-name>ResourceRequestFilter</filter-name>
+              <url-pattern>/*</url-pattern>
+            </filter-mapping>
+          </web-app>
+      ```  
+   -  ```src/main/webapp/WEB-INF/conf/configuration.xml``` : main entry point for configurations available in this extension. This configuration file will loads other configuration files using the import tag. 
+   -  ```src/main/webapp/WEB-INF/conf/custom-extension/portal-configuration.xml``` : Configuration for site metadata configuration, it will add a new navigation link and a new page to the site **DW**
```

Comment:
I'm asking if we should remove the portal configuration in the first step : 
Create the war containing the application, explain how to add it on an existing page. Then in a second step, explaining that we can create a page by configuration, which directly contains the portlet. WDYT ?
It could be more progressive

Reply:
Added a video as requested